### ISSUE TITLE
Warm the users when a valid app url that is not an official safe app is used

### ISSUE THREAD

**Issue by JagoFigueroa** [gnosis/safe-react-apps#430]
## Description

We should warm the users if an opened app is an "unknown" one. 

When a valid url is used to open an app but this app is not part of the list of official apps or a custom one added by the user (with the manifest check) a disclaimer should be shown to notify that it is not part of our ecosystem and can lead to undesired issues.

**Comment by katspaugh** [comment]
To make it less annoying for the user, I would show this warning only once per custom app URL (save in the localStorage).

**Comment by JagoFigueroa** [comment]
Development will be done in https://github.com/safe-global/safe-react/issues/3833 sorry for the duplicity Ivan
